x-pack/plugins/ml/public/application/app.tsx

```diff
@@ -78,31 +79,32 @@ const App: FC<AppProps> = ({ coreStart, deps, appMountParams, isServerless, mlFe
     setBreadcrumbs: coreStart.chrome!.setBreadcrumbs,
   };
 
-  const services = useMemo(() => {
+  const services: StartServices = useMemo(() => {
     return {
-      kibanaVersion: deps.kibanaVersion,
-      share: deps.share,
+      cases: deps.cases,
+      charts: deps.charts,
+      contentManagement: deps.contentManagement,
+      dashboard: deps.dashboard,
       data: deps.data,
+      dataViews: deps.data.dataViews,
       dataViewEditor: deps.dataViewEditor,
-      security: deps.security,
-      licenseManagement: deps.licenseManagement,
-      storage: localStorage,
-      embeddable: deps.embeddable,
-      maps: deps.maps,
-      triggersActionsUi: deps.triggersActionsUi,
       dataVisualizer: deps.dataVisualizer,
-      usageCollection: deps.usageCollection,
+      embeddable: deps.embeddable,
       fieldFormats: deps.fieldFormats,
-      dashboard: deps.dashboard,
-      charts: deps.charts,
-      cases: deps.cases,
-      unifiedSearch: deps.unifiedSearch,
-      licensing: deps.licensing,
+      kibanaVersion: deps.kibanaVersion,
       lens: deps.lens,
+      licenseManagement: deps.licenseManagement,
+      maps: deps.maps,
+      presentationUtil: deps.presentationUtil,
       savedObjectsManagement: deps.savedObjectsManagement,
       savedSearch: deps.savedSearch,
-      contentManagement: deps.contentManagement,
-      presentationUtil: deps.presentationUtil,
+      security: deps.security,
+      share: deps.share,
+      storage: localStorage,
+      triggersActionsUi: deps.triggersActionsUi,
+      unifiedSearch: deps.unifiedSearch,
+      usageCollection: deps.usageCollection,
+      uiActions: deps.uiActions,
```

Comment:
Nit - this should go before `unifiedSearch` alphabetically.

Reply:
10 points for finding the deliberate mistake.
Corrected in https://github.com/elastic/kibana/pull/170868/commits/c074629e08e42cdc0817800204f99196753632ee